Solutions/Jamf Protect/Analytic Rules/JamfProtectAlerts.yaml

```diff
@@ -0,0 +1,81 @@
+id: 6098daa0-f05e-44d5-b5a0-913e63ba3179
+name: Jamf Protect - Alerts 
+description: |
+  'Creates an incident based on Jamf Protect Alert data in Microsoft Sentinel'
+severity: High
+status: Available
+requiredDataConnectors:
+  - connectorId: JamfProtect
+    dataTypes: 
+      - jamfprotect_CL
+suppressionDuration: PT5H
+suppressionEnabled: false
+tactics:
```

Comment:
It is mandatory to provide atleast one tactic and relevant technique in an analytical rule

Reply:
Hi @devikamehra 

Not each alert is related to a TTP, looking at other Solutions and Analytic Rules, i see they are also not having a tactic populated. Is this a hard requirement?

Reply:
Hi @txhaflaire,
It would be great to have atleast one assigned to each rule. We can look at exceptions if any.